Mathlib/Topology/Instances/RatLemmas.lean

```diff
@@ -72,7 +72,7 @@ theorem not_secondCountableTopology_opc : ¬SecondCountableTopology ℚ∞ := by
   exact not_firstCountableTopology_opc inferInstance
 
 instance : TotallyDisconnectedSpace ℚ := by
-  clear p q s t
+  clear p s
```

Comment:
Is that still needed?

Reply:
I have not tried deleting that, but the variables are in scope: the new behaviour does not apply to def's and def-adjacent declarations.

Reply:
I went back and looked at removing `clear` and it is not possible without more changes.  I did not investigate all the way, but it seems like the `wlog` step a little further down generates an implication that uses `p` and `s`, if they are in the local context.